securedrop/source_app/main.py

```diff
@@ -143,19 +163,16 @@ def submit():
             if msg and not fh:
                 html_contents = gettext('Thanks! We received your message.')
             elif not msg and fh:
-                html_contents = gettext('Thanks! We received your document.')
+                html_contents = gettext('Thanks! We received your'
```

Comment:
A source who has only submitted a single file might be concerned when told that "document(s)" have been received.

Might `len(fh) == 1` and `len(fh) >= 1` be used in more `elif` statements to send the appropriate response?